repo2docker/detectors.py

```diff
@@ -520,18 +528,18 @@ def setup_assembly(self):
         # will be installed in python3 venv. This is less of a
         # surprise than requiring python2 to be requirements2.txt tho.
         try:
-            with open('runtime.txt') as f:
+            with open(self.binder_path('runtime.txt')) as f:
                 runtime = f.read().strip()
         except FileNotFoundError:
             runtime = 'python-3.5'
         if runtime == 'python-2.7':
-            requirements_file = 'requirements3.txt'
+            requirements_file = self.binder_path('requirements3.txt')
```

Comment:
This looks like a typo, python-2.7 leading to reading requirements3.txt ?

Reply:
Could you open an issue to discuss this? I also think this is a bit puzzling/should have a comment explain the thinking